resources/dicts/events/death/beach/apprentice.json

```diff
@@ -1,3 +1,225 @@
 [
-
+    {
+        "camp": "any",
+        "tags": [
+            "Newleaf", 
+            "Greenleaf", 
+            "Leaf-fall"
+        ],
+        "death_text": "m_c had not realized {PRONOUN/m_c/poss} fatal mistake until {PRONOUN/m_c/subject} felt {PRONOUN/m_c/poss} paws get clammy and {PRONOUN/m_c/poss} blood, once burning hot and dry after being out in the sun with no water nor shade, turned cold. As the world turned right-side up and {PRONOUN/m_c/poss} consciousness faded right before {PRONOUN/m_c/poss} eyes, m_c knew {PRONOUN/m_c/subject} {VERB/m_c/were/was}beyond saving, as dehydration and overheating claimed the last bits of {PRONOUN/m_c/poss} strength.",
+        "history_text": {
+                        "reg_death": "m_c succumbed to dehydration and overheating."
+                        },
+        "cat_trait": null,
+        "cat_skill": null,
+        "other_cat_trait": null,        
+        "other_cat_skill": null
+     },
+     {
+        "camp": "any",
+        "tags": [
+            "Newleaf", 
+            "Greenleaf", 
+            "Leaf-fall",
+            "other_cat",
+            "other_cat_mentor"
+        ],
+        "death_text": "r_c noticed m_c hadn't shown up for training again, yet after further prodding realized {PRONOUN/m_c/subject} hadn't been at camp for a few days. However, r_c never expected to find m_c like this: dead and dried out, round black beetles crawling in, out, and across {PRONOUN/m_c/poss} once lively pelt.",
```

Comment:
We should avoid any "gross" descriptions of decay or gore. Community feedback has been negative of such events. 

I suggest removing the descriptions of insects crawling over the corpse. I think "dried out" is okay, but people may bulk at insect imagery. 

Suggested:
"r_c noticed m_c hadn't shown up for training again, yet after further prodding realized {PRONOUN/m_c/subject} hadn't been at camp for a few days. However, r_c never expected to find m_c like this: dead and dried out, their tattered pelt lifeless and dull."